src/index.js

```diff
@@ -0,0 +1,20 @@
+const API_KEY = "aaebd4178fcf227a3349fb75ef211df7";
+const API_URL = "https://api.openweathermap.org/data/2.5/weather/?units=metric&q=";
+
+const searchBox = document.querySelector(".search input");
+const searchBtn = document.querySelector(".btn");
+
+async function checkWeather (city) {
+    const response = await fetch(API_URL + city+  `&appid=${API_KEY}`);
+    var data = await response.json();
+
+    document.querySelector(".city").innerHTML = data.name;
+    document.querySelector(".temp").innerHTML = Math.round(data.main.temp) + "°C";
```

Comment:
use template literal for string concatenation

Reply:
Fixed